ebonyspoem

```diff
@@ -0,0 +1,25 @@
+Two memes diverged in a dank montage,
+And sorry I could not watch both
+And be one memer, long I stood
+And looked down one as far as I could
+To where it memed in the undergrowth;
+
+Then took the other, as just as dank,
+And having perhaps the better meme,
+Because it was dank and wanted memes;
+Though as for that the meming there
+Had danked them really about the same,
+
+And both that montage equally lay
+In leaves no step had trodden african american.
+Oh, I kept the first for another day!
+Yet knowing how way leads on to way,
+I doubted if I should ever come back to 9gag.
+
+I shall be telling this with a sigh
+Somewhere ages and ages hence: ******* kiddies
+Two memes diverged in a montage, and I—
+I took the one less memed by,
+And that has made all the dankness.
+
+-John Cena, 2015
```

Comment:
Very nice.  Can you use 2 dashes please?

Reply:
ok